account_banking_mandate/models/account_move.py

```diff
@@ -21,7 +21,8 @@ class AccountMove(models.Model):
     )
 
     def post(self):
-        self.line_ids.mandate_id = self.mandate_id
+        for sel in self:
+            sel.line_ids.mandate_id = sel.mandate_id
```

Comment:
No need of the loop nor a mapped on relational fields in v13:

```suggestion
        self.line_ids.write{"mandate_id": sel.mandate_id}
```

Reply:
HI @pedrobaeza I was trying, i think it needs to loop, because if you select 10 account.moves with diferent mandate in each one. It have to look for each one and write in the specific account.move. Is not the same for all the records.
Then the first code is good or:

    def post(self):
        for sel in self:
            sel.line_ids.write({"mandate_id": sel.mandate_id})
        return super(AccountMove, self).post()

What do you think?

Reply:
Yeah, totally, forget my distraction. Maybe it's better to use a more distinguishable variable name, but yours is correct:
```python
for record in self:
    record.line_ids.write({"mandate_id": record.mandate_id})
```